src/jobsautoreport/slack.py

```diff
@@ -0,0 +1,131 @@
+            {
+                "type": "header",
+                "text": {
+                    "type": "plain_text",
+                    "text": "Jobs Weekly Report",
+                    "emoji": True,
+                },
+            },
+            {
+                "type": "section",
+                "fields": [
+                    {
+                        "type": "mrkdwn",
+                        "text": f"*Jobs success rate*: {self.data['success_rate']}",
+                    }
+                ],
+            },
+            {
+                "type": "section",
+                "fields": [
+                    {
+                        "type": "mrkdwn",
+                        "text": f"*Number of jobs triggered*: {self.data['number_of_jobs_triggered']}",
+                    }
+                ],
+            },
+            {
+                "type": "section",
+                "fields": [
+                    {
+                        "type": "mrkdwn",
+                        "text": f"*Average duration*:{self.data['average_jobs_duration']}",
+                    }
+                ],
+            },
+        ]
+
+        return blocks
+
+    def format_as_comment(self) -> list:
+        blocks = [
+            {
+                "type": "header",
+                "text": {
+                    "type": "plain_text",
+                    "text": "Top 10 failing e2e periodic jobs",
+                    "emoji": True,
+                },
+            },
+            {
+                "type": "section",
+                "fields": [
+                    {"type": "mrkdwn", "text": f"1. Job name: {self.data[0].name}"},
+                    {"type": "mrkdwn", "text": f"Job score: {self.data[0].score}"},
+                ],
+            },
+            {
+                "type": "section",
+                "fields": [
+                    {"type": "mrkdwn", "text": f"2. Job name: {self.data[1].name}"},
+                    {"type": "mrkdwn", "text": f"Job score: {self.data[1].score}"},
+                ],
+            },
+            {
+                "type": "section",
+                "fields": [
+                    {"type": "mrkdwn", "text": f"3. Job name: {self.data[2].name}"},
+                    {"type": "mrkdwn", "text": f"Job score: {self.data[2].score}"},
+                ],
+            },
+            {
+                "type": "section",
+                "fields": [
+                    {"type": "mrkdwn", "text": f"4. Job name: {self.data[3].name}"},
+                    {"type": "mrkdwn", "text": f"Job score: {self.data[3].score}"},
+                ],
+            },
+            {
+                "type": "section",
+                "fields": [
+                    {"type": "mrkdwn", "text": f"5. Job name: {self.data[4].name}"},
+                    {"type": "mrkdwn", "text": f"Job score: {self.data[4].score}"},
+                ],
+            },
+        ]
+
+        return blocks
+
+
+class SlackMessenger:
```

Comment:
I think `SlackMessage`/`SlackMessager` can be confusing cause their names are very close.

Maybe `SlackSender` or `SlackReporter` instead of `SlackMessager`